usr/bin/i2pbrowser

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+##
+##
+
+export SCRIPTNAME="$(basename "$BASH_SOURCE")"
+
+ICON="/usr/share/icons/anon-icon-pack/tbupdate.ico"
+
+if [ ! -d "$HOME/.i2pb/" ]; then
```

Comment:
By using this folder you'll be missing out on

* https://github.com/Whonix/apparmor-profile-torbrowser
* firejail - https://community.parrotsec.org/t/firejail-implementation-details/2175

due to that path.

Not a blocker but may be easier to sort this out now than to migrate the folder later.

Reply:
I noticed that. My plan was just to create the apparmor profile and submit it to the apparmor profile repo if that's fine with you.

Reply:
Better if we could handle it using wildcards. Like this:

https://github.com/Whonix/apparmor-profile-torbrowser/blob/master/etc/apparmor.d/home.tor-browser.firefox#L4

```
@{TBB} = @{HOME}*

/home/**/tor-browser*/Browser/firefox flags=(attach_disconnected) {
```

Might even already match?